Topics/Tech_Stacks/Apache_Kafka.md

```diff
@@ -0,0 +1,102 @@
+# Learning Apache Kafka
```

Comment:
It may be better to keep the title consistent with the one you included in Tech_Stacks.md: "Getting Started with Apache Kafka".

Reply:
Got it, thanks for your suggestion